### ISSUE TITLE
MU_loss

### ISSUE THREAD

**Issue by YuTian315** [xidizxc/CDBNE#2]
我看到run.py里面计算MU_loss的时候是把tensor做了detach(), 然后转成numpy计算的，这样能根据这个MU_loss进行优化吗？

**Comment by Roiko97** [comment]
作者好：MU_loss = Modula(adj.detach().numpy(), A_pred.detach().numpy())这行代码无法优化模型，因为它已经无法进行反向传播了。

**Comment by wx-zhou** [comment]
根据论文中对于Pre-training的描述，我写了一下这块的代码，但是效果很差，作者能提供一下这块的代码吗，谢谢！
According to the description of Pre-training in the paper, I wrote the code for this piece, but the effect is very poor. Can the author provide the code for this piece, thank you!

**Comment by suikai** [comment]
各位都能直接运行吗，是不是少了gae模块啊

**Comment by suikai** [comment]
> 
能直接运行吗，是不是少了gae模块啊

**Comment by Enjoy0223** [comment]
> 各位都能直接运行吗，是不是少了gae模块啊

您好，请问您解决这个问题了吗

**Comment by LuoXinyue217** [comment]
> 是
是的我也找不到gae
